bundle/manifests/multiclusterhub-operator.clusterserviceversion.yaml

```
@@ -571,7 +580,7 @@ spec:
           control-plane: multiclusterhub-operator
         name: multiclusterhub-operator
         spec:
-          replicas: 2
+          replicas: 1
```

We don't want to reduce the replicas count to `1`.
```suggestion
          replicas: 2
```

---

I think I might need some guidance then on operator-sdk, because that change happened when I ran `make bundle`